app/routes/customer_routes.py

```diff
@@ -0,0 +1,133 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request, abort, make_response
+
+from app.routes.video_routes import check_for_valid_input
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+def validate_customer_input(request_body):
+    '''ensure all input from request body is a correct 
+    and valid data type, abort and return 400 error 
+    message if the data is not valid'''
+    for key, value in request_body.items():
+        if type(value) is not str:
+            abort(make_response({"Invalid Input": f"The {key} value must be a string."}, 400))
```

Comment:
love that yall tried something new and thought outside the box to find a solution. one thing I can say is so our particular web API, we want it to always return JSON data. `abort` returns HTML or an HTML page, which would be an unexpected response body for a client.

I think what we could do instead is have this helper function return a boolean, True if everything is valid, False is something has gone wrong. Then we let the route itself handle the response body based on the boolean returned